content/community/news/general/2021-11-10-interchange-update.md

```diff
@@ -0,0 +1,424 @@
+
+```python
+from openff.toolkit.topology import Molecule, Topology
+from openff.toolkit.typing.engines.smirnoff import ForceField
+
+caffeine = Molecule.from_smiles("CN1C=NC2=C1C(=O)N(C(=O)N2C)C")
+caffeine.generate_conformers(n_conformers=1)
+topology = caffeine.to_topology()
+
+sage = ForceField("openff_unconstrained-2.0.0.offxml")
+```
+
+To belabor the point about where exactly in a workflow we are focused, one can load in another [available force field](https://open-forcefield-toolkit.readthedocs.io/en/latest/api/generated/openff.toolkit.typing.engines.smirnoff.forcefield.get_available_force_fields.html#openff.toolkit.typing.engines.smirnoff.forcefield.get_available_force_fields) or generate a prepare a topology with an arbitrary number of different molecules.
+
+
+## Using Interchange
+
+Now that everything is prepared, we simply construct an ``Interchnage`` object with two lines of code:
+
+```python
+from openff.interchange.components.interchange import Interchange
+
+out = Interchange.from_smirnoff(force_field=sage, topology=topology)
+out
+```
+    Interchange with 24 atoms, non-periodic topology
+
+This object is currently missing positions and box information. Let's use the conformer we generated
+earlier and assign box vectors to something more than twice the cutoff that Sage uses (0.9 nanometers).
+
+```python
+out.box = [4, 4, 4]
+out.positions = caffeine.conformers[0]
+out.box
+```
+    array([[4., 0., 0.],
+       [0., 4., 0.],
+       [0., 0., 4.]]) <Unit('nanometer')>
+
+(This step may be more streamlined in the future, possibly with changes directly in the toolkit!)
+
+For more complex systems, there may be other steps involved at different points in a workflow, such
+as adding solvent and counter-ions or combining multiple ``Interchange`` objects generated from
+different force fields. But for this example, we're (partially) done! The ``Interchange`` object is
+fully constructed and we can export its contents to objects understood by simulations engines.
+
+As a brief aside,
+[migrating](https://openff-interchange.readthedocs.io/en/latest/using/migrating.html#migrating-from-the-openff-toolkit)
+existing code from using the OpenFF Toolkit's OpenMM interface to using Interchange directly is
+straightforward. Wherever
+[`ForceField.create_openmm_system](https://open-forcefield-toolkit.readthedocs.io/en/latest/api/generated/openff.toolkit.typing.engines.smirnoff.forcefield.ForceField.html#openff.toolkit.typing.engines.smirnoff.forcefield.ForceField.create_openmm_system)
+is was used, simply replace it with `Interchange.from_smirnoff` like so:
+
+```python
+# Existing code generating an OpenMM System
+openmm_system = forcefield.create_openmm_system(topology)
+
+# New code generating an Interchange object
+out = Interchange.from_smirnoff(forcefield, topology)
```

Comment:
Maybe have the variable set by this line be called `interchange` to make it less ambiguous